datascience-notebook/Dockerfile

```diff
@@ -11,9 +11,16 @@ RUN apt-get update && \
     apt-get install -y --no-install-recommends \
     fonts-dejavu \
     gfortran \
+    unixodbc-dev \
+    libtool-bin \
+    autoconf \
+    automake \
     gcc && apt-get clean && \
     rm -rf /var/lib/apt/lists/*
 
+# Python pyodbc
+RUN pip install pyodbc
```

Comment:
Haven't tried that out but if it works it would be preferred over the non-conda.  Do you have a database to test against?  We should install the conda version for both Python2 and Python3.